var/spack/repos/builtin/packages/py-pyside2/package.py

```diff
@@ -17,6 +17,7 @@ class PyPyside2(PythonPackage):
     # https://wiki.qt.io/Qt_for_Python_Development_Getting_Started
 
     version("develop", tag="dev")
+    version("5.15.10", tag="v5.15.10-lts-lgpl", submodules=True)
```

Comment:
Do you know why this version isn't available on PyPI? I was going to say we should just use PyPI instead of git for downloading

Reply:
I don't know, I just looked for a newer version because I needed it to fulfil version constraints (something else depended on newer qt in my toolchain).
The documentation on available versions of pyside2 for qt5 did not look straight-forward to me...